docs/root/intro/arch_overview/other_protocols/postgres.rst

```diff
@@ -0,0 +1,31 @@
+.. _arch_overview_postgres:
+
+Postgres
+==========
+
+Envoy supports a network level Postgres sniffing filter to add network observability. By using the
+Postgres proxy, Envoy is able to decode `Postgres frontend/backend protocol`_ and gather
+statistics from the decoded information.
+
+The main goal of the Postgres filter is to capture runtime statistics without impacting or
+generating any load on the Postgres upstream server, it is transparent to it. The filter currently
+offers the following features:
+
+* Decode non SSL traffic, ignore SSL traffic.
+* Decode session information.
+* Capture transaction information, including commits and rollbacks.
+* Basic decoding of the incoming SQL, exposing counters for different types of
```

Comment:
In the filter docs you state that SQL is not decoded, but here that it is...

Reply:
Corrected and added explanation that counters are taken from _CommandComplete_ message, not from incoming messages from a client. The real difference is that _CommandComplete_ indicates that command was successful. Incoming message with SQL statement may result in success or error. The next phase of this filter will include parsing SQL in incoming messages and producing metadata similar to mysql.